packages/apollo-language-server/src/schema/providers/index.ts

```diff
@@ -24,6 +24,10 @@ export function schemaProviderFromConfig(
   config: ApolloConfig
 ): GraphQLSchemaProvider {

```

Comment:
We should also have a nice error if this function can't find the needed information to find a service